### PR TITLE
Pelisplus fixes

### PR DESCRIPTION
**Pull request by Stormunblessed**
Now gets sub, dub, castillan, and default server

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusProviderTemplate.kt [hunk 1]
```diff
@@ -156,12 +148,12 @@ open class PelisplusProviderTemplate : MainAPI() {
             val document = Jsoup.parse(response)
             document.select("div.main-inner")?.forEach { inner ->
                 // Always trim your text unless you want the risk of spaces at the start or end.
-                val title = inner.select(".widget-title").text().trim()
+                val title = inner.select(".widget-title").text().replace(Regex("([Tt]emporada (\\d+)|[Cc]apítulo (\\d+))"),"").trim()
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/PelisplusProviderTemplate.kt:151]: it looks like you are using the regex `"([Tt]emporada (\\d+)|[Cc]apítulo (\\d+))"` a lot, I would suggest making it a function

**Comment by LagradOst** [comment]
Just that one thing, then merge

**Comment by LagradOst** [comment]
LGTM, ready to merge? :+1:

**Comment by Stormunblessed** [comment]
> LGTM, ready to merge? 👍

Yes

**Pull request merged**